### PR TITLE
Add support for adding selectively visible messages

### PR DESCRIPTION
**Pull request by rishabhja**

### CODE REVIEW

**Review by chaitanyagupta — COMMENTED** [review]
Please remove the .DS_Store file.

**Review by chaitanyagupta — COMMENTED** [review]
Why have we changed the naming convention from underscores to camel case? @rishabhja

File: flockos/apis/chat.py [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 def fetch_messages(token, chat, uids, **kwargs):
     """
-    
-    
+
+
```
Comment (chaitanyagupta) [inline flockos/apis/chat.py]: This is an unnecessary whitespace change.

**Comment by chaitanyagupta** [comment]
@rishabhja in future, if you want to do things like change naming conventions in a project, do it in a separate PR and not mix it up with other changes.

File: flockos/apis/chat.py (hunk 1 above)
Comment (rishabhja) [reply flockos/apis/chat.py]: Fixed

**Comment by chaitanyagupta** [comment]
@rishabhja this looks fine to me. You can merge this one.

That said, there's one thing that's bugging me. When you compare the complete diff i.e. https://github.com/flockchat/pyflock/pull/13/files to the one which excludes whitespace related changes i.e. https://github.com/flockchat/pyflock/pull/13/files?w=1, why are they different? Do figure out what's happening here.

**Comment by chaitanyagupta** [comment]
@rishabhja this looks fine to me. You can merge this one.

That said, there's one thing that's bugging me. When you compare the complete diff i.e. https://github.com/flockchat/pyflock/pull/13/files to the one which excludes whitespace related changes i.e. https://github.com/flockchat/pyflock/pull/13/files?w=1, why are they different? Do figure out what's happening here.
